src/styles/blocks/main/body.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.page {
+  background-color: $background_white-color;
```

Comment:
Ensure that the variable $background_white-color is defined in your SCSS. If it's not defined, the SCSS will fail to compile. If it's defined in another file, ensure that file is imported correctly.